src/components/HendelseTypeFilter.tsx

```diff
@@ -24,6 +24,7 @@ export const HendelseTekster = {
   BEHANDLER_BER_OM_BISTAND: 'Behandler ber om bistand',
   ARBEIDSUFORHET: '§8-4 Arbeidsuførhet',
   FRISKMELDING_TIL_ARBEIDSFORMIDLING: '§8-5 Friskmelding',
+  SNART_SLUTT_PA_SYKEPENGENE: 'Snart slutt på sykepengene',
```

Comment:
Dette burde også hete `Oppfølging i sen fase`? (Hvor brukes disse tekstene hen egentlig?)

Reply:
Det er teksten som står ved siden av filterne. Stine foreslo "Snart slutt på sykepengene". Tror også veilederne lettere kan gjette seg frem til hva dette er, enn `Oppfølging i sen fase`

Reply:
Hm, men kan "Snart slutt på sykepengene" misforstås siden vi i første omgang bare viser de som har svart og ikke alle som har mindre enn 90 dager igjen? Går jo an å snakke med pilotkontoret om det, så får vi ta en runde på begrepet før vi åpner for flere.

Reply:
De vil nok uansett bruke det gamle systemet parallellt til vi kan vise at vi har noe som fanger alle. Og enig i at vi bør prate med pilotkontoret om begrepet. Regner ikke med at brukerne helt av seg selv skjønner at de som kommer inn her har svart på et skjema når det er et visst antall dager igjen

Reply:
:+1: Mener å huske at eSyfo/Julie også snakket om å gå bort fra å bruke "Snart slutt på sykepengene" som begrep, men det var kanskje mest ut mot bruker og ikke veilederne.

Reply:
Vi hører med de første vi tester mot 😊